commander/src/bootstrapping/templates/lisk-template-ts/templates/init_plugin/src/plugin.ts

```diff
@@ -1,38 +1,14 @@
 import { BasePlugin } from 'lisk-sdk';
-import type { BaseChannel, EventsDefinition, ActionsDefinition, PluginInfo } from 'lisk-sdk';
+import type { BaseChannel } from 'lisk-sdk';
 
  /* eslint-disable class-methods-use-this */
  /* eslint-disable  @typescript-eslint/no-empty-function */
  export class <%= className %> extends BasePlugin {
 	// private _channel!: BaseChannel;
 
-	public static get alias(): string {
-		return '<%= alias %>';
-	}
-
-	// eslint-disable-next-line @typescript-eslint/class-literal-property-style
-	public static get info(): PluginInfo {
-		return {
-			author: '<%= author %>',
-			version: '<%= version %>',
-			name: '<%= name %>',
-		};
-	}
-
-	public get events(): EventsDefinition {
-		return [
-			// 'block:created',
-			// 'block:missed'
-		];
-	}
-
-	public get actions(): ActionsDefinition {
-		return {
-		// 	hello: async () => { hello: 'world' },
-		};
-	}
+	public name:'<%= name %>';
```

Comment:
maybe it would be nice to have 
```
public nodeModulePath = __filename;
```
as well for easy use, and keep `events` and `actions` so that user can start implementing easily